Packs/CortexXDR/Playbooks/Cortex_XDR_-_quarantine_file_v2.yml

```diff
@@ -0,0 +1,180 @@
+id: Cortex XDR - quarantine file v2
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Cortex XDR - quarantine file v2
+description: This playbook accepts file path, file hash and endpoint id in order to quarantine a selected file and wait until the action is done.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 698ce1e6-4540-40bb-8ec4-5dc2decebd93
+    type: start
+    task:
+      id: 698ce1e6-4540-40bb-8ec4-5dc2decebd93
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 1809cbad-2546-4991-8fd5-158ef8df6059
+    type: regular
+    task:
+      id: 1809cbad-2546-4991-8fd5-158ef8df6059
+      version: -1
+      name: Quarantine file
+      description: Quarantine a file on selected endpoints. You can select up to 1000 endpoints.
+      script: '|||xdr-file-quarantine'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      endpoint_id_list:
+        simple: ${inputs.endpoint_id}
```

Comment:
if you can change that to complex instead of simple value that would be great :) 
Same for all simple values in this playbook.
Sidenote - To avoid user errors, we can add `unique` and `first array element` transformers ( up to you).